### PR TITLE
Fixup README and Dockerfiles

### PR DESCRIPTION
**Pull request by domesticmouse**
Fixup Dockerfiles

TBR: @devoncarew

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,13 @@ RUN pub get --offline
 ENV PATH="/home/dart/.pub-cache/bin:${PATH}"
 
 # Set the Flutter SDK up for web compilation.
-RUN dart pub run grinder setup-flutter-sdk
+RUN dart run tool/update_sdk.dart
+
+# Set the Flutter SDK up for web compilation.	# Set the Flutter SDK up for web compilation.
+RUN flutter-sdk/bin/flutter doctor
```
Comment (devoncarew) [inline Dockerfile:31]: We may want to run `flutter config --no-analytics` before we run other flutter commands.

File: README.md [hunk 2]
```diff
@@ -36,12 +36,11 @@ $ pub global activate protoc_plugin
...
 ## Initialise Flutter
 
-The Flutter submodule in the git repo needs to also be downloaded and setup.
+The Flutter SDK needs to be downloaded and setup.
 
 ```bash
-$ git submodule update --init
-$ ./flutter/bin/flutter doctor -v
-$ ./flutter/bin/flutter/bin/flutter config --enable-web
+$ dart pub install
```
Comment (devoncarew) [inline README.md:42]: what does `pub install` do?

File: Dockerfile (hunk 1 above)
Comment (domesticmouse) [reply Dockerfile:31]: These Dockerfiles only get run by the Cloud Build tooling, thus only once per commit. I doubt it's going to change the stats =)

File: README.md (hunk 2 above)
Comment (domesticmouse) [reply README.md:42]: A clean checkout has none of the dart deps downloaded, and `grind deploy` fails in that case. Running `dart pub install` pulls these dependencies, and then `grind deploy` runs clean.

Comment (devoncarew) [reply README.md:42]: Ah, `pub install` is an alias for `pub get` - I wasn't aware of it.

**Pull request merged**